draft-dekater-scion-pki.md

```diff
@@ -1009,7 +1009,7 @@ That is, the quorum defined in the TRC's `votingQuorum` field ([](#quorum)) MUST
 
 A TRC contains policy information about an ISD and acts as a distribution mechanism for the trust anchors of that ISD.
 
-Each TRC is digitally signed and the syntax used to sign and encapsulate the TRC payload is the Cryptographic Message Syntax (CMS) as defined in {{RFC5652}}. The signed TRC payload is of the CMS signed-data content type, as defined in Section 5 of {{RFC5652}}, and encapsulated in a CMS `ContentInfo` element, as defined in Section 3 of {{RFC5652}}.
+Each TRC is digitally signed and the syntax used to sign and encapsulate the TRC payload is the Cryptographic Message Syntax (CMS) as defined in Section 5 of {{RFC5652}}. The signed TRC payload is encapsulated in a CMS `ContentInfo` element as defined in Section 3 of {{RFC5652}}.
 
 For detailed information on the general syntax definitions of the Cryptographic Message Syntax, see sections 3 and 5 of {{RFC5652}}.
 
```

Comment:
I think what Ramon suggested (to reference section 5) was already in the text, but later. The problem is that  RFC5652  is referenced too many times. I think we could improve it: 
```suggestion
Each TRC is digitally signed and the syntax used to sign and encapsulate the TRC payload is the Cryptographic Message Syntax (CMS) as defined in Section 5 of {{RFC5652}}. The signed TRC payload is encapsulated in a CMS `ContentInfo` element as defined in Section 3 of {{RFC5652}}.
```

Reply:
Maybe we could also change the first reference to this RFC: 
```The TRC is a signed collection of [X.509] v3 certificates.
   Additionally, the TRC contains ISD-specific policies encoded in a
   Cryptographic Message Syntax (CMS) [RFC5652] envelope.
   ```